generate/post_process_json.go

```diff
@@ -0,0 +1,24 @@
+package generate
+
+import (
+	"bufio"
+	"bytes"
+	"regexp"
+)
+
+// jsonTagRegexp is the regular express to make the json struct tag to the same name as the one defined in protobuf tag.
+// Given this input: `protobuf:"bytes,1,opt,name=FirstName,json=first_name,proto3" json:"first_name,omitempty"`
```

Comment:
nitpick: should be FirstName in the input in the json tag

Reply:
fixed.